src/App.js

```diff
@@ -1,25 +1,16 @@
-import logo from './logo.svg';
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  render() {
+    return <Calculator />;
+  }
 }
 
+// function App() {
+//   return <Calculator />;
+// }
```

Comment:
## [OPTIONAL]
* I strongly suggest you remove commented code it makes your codebase bulky and unprofessional